integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/addsegment/AddSegmentTestCase.scala

```diff
@@ -722,12 +723,28 @@ class AddSegmentTestCase extends QueryTest with BeforeAndAfterAll {
     val externalSegmentPath = storeLocation + "/" + "external_segment"
     FileFactory.deleteAllFilesOfDir(new File(externalSegmentPath))
 
+    var fields: Array[Field] = new Array[Field](14)
```

Comment:
why is this test case changed?

Reply:
This test is changed just to check for the schema which we give externally instead of referring to schema file of the already existing table.